requirements.txt

```diff
@@ -1,4 +1,5 @@
 boto3
 fmeval
 pandas==2.1.4
-ipywidgets
\ No newline at end of file
+ipywidgets
+pytest
```

Comment:
Shouldn't be here since it's installed by the notebook where tests are irrelevant. I propose to put all test stuff in ./Tests/
And then we'll have ./Tests/requirements.txt
See more [here](https://www.geeksforgeeks.org/typical-directory-structure-for-running-tests-using-unittest-in-python/)